elasticsearch/roles/prion_index_read_phi.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Good news! In the latest release, we have granted DSM full read access (`pepper_index_read_phi`) to data in Elasticsearch, so I don't think we'll need a separate role for grant read on prion data.